GA/HeaderFiles/DataRead.h~

```diff
@@ -0,0 +1 @@
+#pragma once
```

Comment:
Is this file supposed to be included?

If not, you can add files ending with `~` to the `.gitignore` so that they don't get added accidentally and/or use `git add <filenames>` explicitly, `git add -p`, and `git status` before committing.